.gitignore

```diff
@@ -4,3 +4,4 @@
 /src/framework/preferences/css/*.css
 /.bundle/
 /.vagrant/
+report.tap
```

Comment:
Does this mean it will leave  "report.tap" file in the root directory? Is there a way to clean this up?